agent/config/config.go

```diff
@@ -248,6 +250,18 @@ func EnvironmentConfig() Config {
 		}
 	}
 
+	dockerStopTimeoutSecondsEnv := os.Getenv("ECS_CONTAINER_STOP_TIMEOUT")
+	var dockerStopTimeoutSeconds uint64
+	if dockerStopTimeoutSecondsEnv != "" {
+		dockerStopTimeoutSeconds, err = strconv.ParseUint(dockerStopTimeoutSecondsEnv, 10, 64)
```

Comment:
Rather than using `strconv.ParseUint` here, I think it'd be better to use [`time.ParseDuration`](https://golang.org/pkg/time/#ParseDuration).